src/enhancers/exportDLJSInterface.ts

```diff
@@ -48,8 +49,14 @@ export default function exportDLJSInterface<TAdapterState extends AdapterState>(
 
     adapter.addEventListener('open', async () => {
       if(!connectionStatusObserver){
-        let waitTime = 5;
-        while(!connectionStatusObserver && waitTime < 2000){
+        adapter.getState(StateKey.Logger)?.logClientSdkTelemetryEvent(Microsoft.CRM.Omnichannel.IC3Client.Model.LogLevel.ERROR,
```

Comment:
Is this supposed to be Error? If this is possible/not fatal should we change it to WARN?

Reply:
good catch.